src/main/resources/db/migration/R__TypeCodes.sql

```diff
@@ -208,6 +208,12 @@ VALUES (1, 'Seed collection & germination'),
        (7, 'Other')
 ON CONFLICT (id) DO UPDATE SET name = excluded.name;
 
+INSERT INTO wood_density_levels (id, name)
+VALUES (1, 'Species level'),
+       (2, 'Genus level'),
+       (3, 'Family level')
```

Comment:
What would you think of removing "level" from these names, such that the enum names in the code look like `WoodDensityLevel.Species` instead of `WoodDensityLevel.SpeciesLevel`? The strings file can still include the word "level" in the English display name.

Definitely not important; it just felt a little redundant to have "Level" twice in close succession.

Reply:
Good call